docs/source/faq/index.md

```diff
@@ -2,10 +2,10 @@
 
 ### How do I implement paging?
 
-When using [native protocol](../manual/native_protocol/) version 2 or
+When using [native protocol](../manual/native_protocol/index) version 2 or
```

Comment:
This fix are problematic, since it will make sync with upstream very hard, and we would like to sync with upstream often.
It would be much better to write a small code that does this automatically.

Reply:
I'm thinking about how we can add Sphinx without modifying much the folder structure/links so that you can pull from the origin without having too many conflicts. I hope to submit a fix by the end of the week.

Reply:
I've noticed that adding a custom script that reads all files and adds ``index`` to every relative link has one caveat. Differently to the upstream project, every Scylla doc project comes with Sphinx support and commands to build and preview the docs.  If the custom script is executed before previewing the docs, this changes all contents. Most likely, this would make the next documentarian editing the docs to commit all the edited links by mistake. Contrarily, if the custom script is not executed before the preview, it will not be possible to preview the docs using Sphinx due to the errors raised.

**Recommendation** 

Unlike GitHub, Sphinx requires to pass the complete path to the file. Before over-architecting a solution or losing functionality, I'd opt to merge the PR as is. After pulling from upstream the next version, we could evaluate if the number of links affected makes the solution not manageable. For instance, after the recent ``force push`` to the latest branch, only the changelog and a couple of links were affected.

**Other option tested**

- Change the Markdown Parser: Tested other Markdown parsers like Myst and recommonmark versions. All of them keep raising the same build errors.

- Overwrite relative links behavior:  I rejected this alternative because of the ``node.references()`` function is too attached to the core implementation of Sphinx and docutils. 

- Custom directive: Adding a separate directive would not solve the requirements since it would require to edit every line affected to call the new directive.